wkt/src/parsing.py

```diff
@@ -0,0 +1,154 @@
+from functools import singledispatch
+from typing import Union
+
+from shapely import wkt, GEOSException
+from shapely.geometry import (
+    Point,
+    LineString,
+    Polygon,
+    MultiPoint,
+    MultiLineString,
+    MultiPolygon,
+)
+from shapely.geometry.base import BaseGeometry
+
+from traveltimepy import Coordinates
+from wkt.src import (
```

Comment:
Good point, moved wkt to traveltimepy. For now placed tests together with all other tests, but it might make sense to create a sub-dir in /tests for the wkt tests.